app/views/works/top.html.erb

```diff
@@ -0,0 +1,39 @@
+<br>
+<section>
+  <% if !Work.spotlight.nil? %>
+<h2>
+    <span class="spotlight"> Media Spotlight: </span>
+  <% top_work = Work.spotlight %>
+  <%= link_to top_work.title, work_path(top_work.id)%> by <%= top_work.creator %>
+  </h2>
+    <% end %>
+</section>
+
+<br>
```

Comment:
`<br>` is not a semantic tag-- it deals with how the page looks (it makes a line break). If you wanted space between the sections, I would add top or bottom margin! %>